osu.Game.Rulesets.Catch.Tests/Editor/TestSceneJuiceStreamSelectionBlueprint.cs

```diff
@@ -158,14 +158,14 @@ public void TestAddVertex()
             float[] positions = { 200, 200 };
             addBlueprintStep(times, positions, 0.2);
 
-            addAddVertexSteps(500, 150);
-            addVertexCheckStep(3, 1, 500, 150);
+            addAddVertexSteps(500, 180);
+            addVertexCheckStep(3, 1, 500, 180);
 
             addAddVertexSteps(90, 200);
             addVertexCheckStep(4, 1, times[0], positions[0]);
 
-            addAddVertexSteps(750, 180);
-            addVertexCheckStep(5, 4, 750, 180);
+            addAddVertexSteps(750, 200);
+            addVertexCheckStep(5, 4, 750, 200);
```

Comment:
If these test "fixes" appear arbitrary - that is because they are. They would fail after the rest of the changes, and as far as I can tell that is just a manifestation of https://github.com/ppy/osu/issues/28915 that wasn't there before due to the lack of inline `UpdateHitObjectFromPath()` calls. I would ask to look away from this for now while I attempt to figure out what is going on over there (it is not easy).